### PR TITLE
Add details about dependency version dropping PR rule

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

File: project/CONTRIBUTING.md [hunk 1]
```diff
@@ -324,6 +324,9 @@ but some extra rules must be respected here:
   - PHP versions that are under the [green zone][php_supported_versions] (actively maintained) **MUST NO** be dropped, even on master.
   - If it's a Symfony package, at least the last LTS version **MUST** be supported, even on master.
   - Generally, don't drop dependency version it it doesn't have a big impact on the code.
+  - Backward Compatible code related to the dropped version **MUST** be dropped on the same PR.
+    This will permit to see if this version dropping **really worth it** or not.
```
Comment (jordisala1991) [inline project/CONTRIBUTING.md]: *is really worth it?

Comment (Soullivaneuh) [reply project/CONTRIBUTING.md]: "It's really worth it! Go!"

Indeed. See? Your English is not that bad! :stuck_out_tongue:

File: project/CONTRIBUTING.md [hunk 2]
```diff
@@ -325,6 +325,9 @@ but some extra rules must be respected here:
   - PHP versions that are under the [green zone][php_supported_versions] (Active Support) **MUST NOT** be dropped on the master branch.
   - If it's a Symfony package, at least the last LTS version **MUST** be supported, even on master.
   - Generally, don't drop dependency version it it doesn't have a big impact on the code.
+  - Backward Compatible code related to the dropped version **MUST** be dropped on the same PR.
+    This will permit to see if this version dropping **is really worth it** or not.
```
Comment (greg0ire) [inline project/CONTRIBUTING.md]: permit => allow
dropping => dorp

Comment (Soullivaneuh) [reply project/CONTRIBUTING.md]: dorp? derp?! :trollface:

File: project/CONTRIBUTING.md [hunk 3]
```diff
@@ -325,6 +325,9 @@ but some extra rules must be respected here:
...
+    Please note that we can refuse a version dropping at any moment if the change gain is not enough.
```
Comment (greg0ire) [inline project/CONTRIBUTING.md]: "version drop"
"if the gain does not seem sufficient"

**Pull request merged**